src/Components/SearchResultsContainer/SearchResultsContainer.css

```diff
@@ -1,31 +1,31 @@
 .search-results-container {
-    width: 80%;
-    margin: 2% auto;
-    display: flex;
-    flex-direction: row;
-    align-items: flex-start;
+  width: 80%;
+  margin: 2% auto;
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  visibility: hidden;
```

Comment:
Since there's no functionality I decided to remove it to avoid confusion entirely.

Reply:
I'm using `visibility: hidden` instead of `display: none` to keep this element in the DOM - I want the rest of the formatting to stay the same.